### PR TITLE
updated docs

### PR DESCRIPTION
**Pull request by tjperry07**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,40 +62,33 @@ There are four configs within the Work with Task Options group.
...
+		* UNION ALL - Combines without duplicate elimination.
     * False - Single source node or multiple sources combined using a join.
-* **Cluster key**: True/False to determine whether Dimension is to be clustered or not
-    * True -Allows you to specify the column based on which clustering is to be done.
-            -Allow Expressions Cluster Key-True ->allows to add an expression to the specified cluster key
+* **Cluster key**: True/False to determine whether Dimension is to be clustered or not.
+    * True - Allows you to specify the column based on which clustering is to be done.
+	    *  **Allow Expressions Cluster Key**: True/False.  Add an expression to the specified cluster key.
     * False – No clustering done
+ * **Truncate Before**: [Missing]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn

File: README.md [hunk 2]
```diff
@@ -109,505 +99,538 @@ There are 5 to 7 options that can be selected depending on combinations of confi
...
     * False - Single source node or multiple sources combined using a join.
-* **Business key**: It is a required column for Fact table creation.
-* **Cluster key **: True/False to determine whether Fact is to be clustered or not
-    * True -Allows you to specify the column based on which clustering is to be done.
-            -Allow Expressions Cluster Key-True ->allows to add an expression to the specified cluster key
+* **Cluster key**: True/False to determine whether Dimension is to be clustered or not.
+    * True - Allows you to specify the column based on which clustering is to be done.
+	    *  **Allow Expressions Cluster Key**: True/False.  Add an expression to the specified cluster key.
     * False – No clustering done
+ * **Truncate Before**: [Missing]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn

File: README.md [hunk 3]
```diff
@@ -718,52 +747,48 @@ The Coalesce Stream UDN is a node that allows you to develop and deploy a stream
...
+
+* **Source Object**: The type of object the stream will be created on. One of three options is required to be selected and this selection drives what other configs are available:
+    * **Table**
+	    * **Append Only Stream**: True / False Toggle to set what type of stream is created.
+		    * True - Specifies an append-only stream.
+		    * False - Specifies a standard stream.
+	    * **Show Initial Rows**: True / False Toggle to specify the records to return the first time the stream is consumed.
+			* True - The stream returns only the rows that existed in the source object at the moment when the stream was created. Subsequently, the stream returns any DML changes to the source object since the most recent offset; that is, the normal stream behavior.
+			* False - The stream returns any DML changes to the source object since the most recent offset.
+		* **Redeployment Behavior**: [Missing content]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn

File: README.md [hunk 4]
```diff
@@ -718,52 +747,48 @@ The Coalesce Stream UDN is a node that allows you to develop and deploy a stream
...
+    * **View**:
+	   * **Append Only Stream**: True / False Toggle to set what type of stream is created.
+		    * True - Specifies an append-only stream.
+		    * False - Specifies a standard stream.
+		* **Show Initial Rows**: True / False Toggle to specify the records to return the first time the stream is consumed.
+		    * True - The stream returns only the rows that existed in the source object at the moment when the stream was created. Subsequently, the stream returns any DML changes to the source object since the most recent offset; that is, the normal stream behavior.
+			* False - The stream returns any DML changes to the source object since the most recent offset.
+		* **Redeployment Behavior**: [Missing content]
+    * **External Table**: [Missing content. When I change to external table in the UI, it only shows redeployment behavior]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn

File: README.md [hunk 5]
```diff
@@ -856,60 +878,30 @@ The Stream has two configuration groups:
...
+
+* **Source Object**: The type of object the stream will be created on. One of three options is required to be selected and this selection drives what other configs are available:
+    * **Table**
+	    * **Append Only Stream**: True / False Toggle to set what type of stream is created.
+		    * True - Specifies an append-only stream.
+		    * False - Specifies a standard stream.
+	    * **Show Initial Rows**: True / False Toggle to specify the records to return the first time the stream is consumed.
+			* True - The stream returns only the rows that existed in the source object at the moment when the stream was created. Subsequently, the stream returns any DML changes to the source object since the most recent offset; that is, the normal stream behavior.
+			* False - The stream returns any DML changes to the source object since the most recent offset.
+		* **Redeployment Behavior**: [Missing content]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn

File: README.md [hunk 6]
```diff
@@ -856,60 +878,30 @@ The Stream has two configuration groups:
...
+    * **View**:
+	   * **Append Only Stream**: True / False Toggle to set what type of stream is created.
+		    * True - Specifies an append-only stream.
+		    * False - Specifies a standard stream.
+		* **Show Initial Rows**: True / False Toggle to specify the records to return the first time the stream is consumed.
+		    * True - The stream returns only the rows that existed in the source object at the moment when the stream was created. Subsequently, the stream returns any DML changes to the source object since the most recent offset; that is, the normal stream behavior.
+			* False - The stream returns any DML changes to the source object since the most recent offset.
+		* **Redeployment Behavior**: [Missing content]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn

File: README.md [hunk 7]
```diff
@@ -856,60 +878,30 @@ The Stream has two configuration groups:
...
+    * **External Table**: [Missing content. When I change to external table in the UI, it only shows redeployment behavior]
```
Comment (tjperry07) [inline README.md]: @jgontarz @prj2001-udn

File: README.md (hunk 1 above)
Comment (prj2001-udn) [reply README.md]: MIssing content added

**Review by prj2001-udn — COMMENTED** [review]
Changes reviewed and updates already resolved

**Pull request merged**